src/testjfxapp/AccesSettings.java

```diff
@@ -0,0 +1,50 @@
+package testjfxapp;
+// Work in progress
+
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+public class AccesSettings {
+
+    Stage window;
+    Scene accesSettings;
+    Button enterSettings, enterScoreboard, sp, mp;
+    Button btm1, btm2, btm3;
+    Label mainMenuLabel;
+
+    MainMenu mainMenu;
+
+    public AccesSettings(MainMenu m){
+        mainMenu = m;
+    }
+
+   // @Override
+    public void start(Stage primaryStage) throws Exception {
+
+        window = primaryStage;
+
+        btm1 = new Button("Back to Menu");
+        btm1.setOnAction(e -> window.setScene(mainMenu.mainMenu));
+
+//        btm2 = new Button("Back to Menu");
+//        btm2.setOnAction(e -> window.setScene(settings));
+//
+//        btm3 = new Button("Back to Menu");
+//        btm3.setOnAction(e -> window.setScene(settings));
```

Comment:
An idea might be to have the previous menu classes pass themselves to all new menus, then have a function within them that sets the current scene to their scene.
An idea to implement this would be to have every settings menu extend a SettingsMenu class, then have that class have a function to set the window scene to its scene, and have each sub-menu have a constructor that takes in a SettingsMenu object (which, assuming they extend said SettingsMenu class, should be able to be done with any given menu), have the object store the previous settings menu and have the back buttons call that menu's scene setting function.